website/static/js/project-organizer.js

```diff
@@ -53,7 +53,10 @@ function _poTitleColumn(item) {
  * @private
  */
 function _poContributors(item) {
-    var contributorList = item.data.embeds.contributors.data;
+    var contributorList = [];
+    if (item.data && item.data.embeds && item.data.embeds.contributors) {
+        contributorList = item.data.embeds.contributors.data;
+    }
     if(!contributorList){
```

Comment:
due to your changes, this will never return `''`, since `[]` evaluates to ` true `. This is taken of by line 63:
```
if (contributorList.length === 0) {
         return '';
     }
```, so I think this `if` can be removed.